lessons/06-apis-debugging/README.md

```diff
@@ -1,9 +1,13 @@
 #Lesson 6 - APIs and Debugging Strategies
+##Preclass 
+Please read the following Readme and some of the following links on [Oauth](http://www.cubrid.org/blog/dev-platform/dancing-with-oauth-understanding-how-authorization-works/) and [REST](http://www.infoq.com/articles/rest-introduction)
+
 ##APIs: Your *real* on-ramp to the information superhighway
+*Author: Evan Simpson 2015*
```

Comment:
Unless you're citing previous authors for a specific reason (or doing it consistently), it's probably cleaner to leave it out.